tests/unit-tests/geolocation/class-wc-tests-geolite-integration.php

```diff
@@ -1,40 +0,0 @@
-<?php
-/**
- * Class Functions.
- *
- * @package WooCommerce\Tests\Geolocation
- */
-
-/**
- * Class WC_Tests_Integrations
- */
-class WC_Tests_Geolite_Integration extends WC_Unit_Test_Case {
-
-	/**
-	 * Test get country ISO.
-	 *
-	 * @requires PHP 5.4
-	 */
-	public function test_get_country_iso() {
-		require_once dirname( WC_PLUGIN_FILE ) . '/includes/class-wc-geolite-integration.php';
-
-		// Download GeoLite2 database.
-		WC_Geolocation::update_database();
-
-		// OpenDNS IP address.
-		$ipv4 = '208.67.220.220';
-		$ipv6 = '2620:0:ccc::2';
-
-		// Init GeoLite.
-		$geolite = new WC_Geolite_Integration( WC_Geolocation::get_local_database_path() );
-
-		// Check for IPv4.
-		$this->assertEquals( 'US', $geolite->get_country_iso( $ipv4 ) );
-
-		// Check for IPv6.
-		$this->assertEquals( 'US', $geolite->get_country_iso( $ipv6 ) );
-
-		// Check for non-valid IP.
-		$this->assertEquals( '', $geolite->get_country_iso( 'foobar' ) );
-	}
-}
```

Comment:
Since it's deprecated we should not update that class anymore, also there's no reason for use this class anymore.
But I agree that we could keep this unit tests in place while we don't remove that class from core.

Reply:
Ohh, I just checked and the current code makes hard to keep this unit test working, currently this test is failing, since it's not possible to download the database without a license key, also that method uses MaxMind Reader library to read the file, we currently have another tests for it too, and if we move that code to read a custom database in our unit tests will not make any sense to keep this test, since is more like testing the MaxMind Reader Library than code on WooCommerce.

Reply:
This was my thinking in removing the test.

Reply:
Since the geo database is now included with the tests in `tests/data`, does it matter that the license code is not there? I thought the class should still continue to work even if it's deprecated, no?

I agree there is not that much of our code to test, but the code for the method `get_country_iso` hasn't changed, so why it made sense to test it before, but it doesn't make sense to test it now? I still think it's good to have the test to make sure we don't break bw compat of this class until we decide to delete it.

Reply:
The included database isn’t actually real. We can’t include the GeoLite database with the repository, it’s an empty file for testing the service.

Reply:
The included database isn’t actually real. We can’t include the GeoLite database with the repository, it’s an empty file for testing the service.

Reply:
ah, I see. Haven't realized that, apologies. Let's remove it, then.